plugin-server/src/main/ingestion-queues/session-recording-v2/sessions/session-batch-file-format.integration.test.ts

```diff
@@ -174,7 +175,9 @@ describe('session recording integration', () => {
         ]
 
         // Record all messages
-        messages.forEach((message) => recorder.record(message))
+        for (const message of messages) {
+            await recorder.record(message)
+        }
```

Comment:
issue: `record` is not an async function, it should not be called with `await`. It's synchronous by design, so that we don't have any I/O in the middle of the batch.